src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import './styles/dictionary-modal.css';
+
+const DictionaryDetailCard = dictionary => {
```

Comment:
Brackets enclosing the dictionary parameter are missing

Reply:
Done